### PR TITLE
Same returns and updated Docs

### PR DESCRIPTION
**Pull request by sjfricke**
I added `return this;` to all methods except the few boolean `ready()` ones. I also added `@return` for JDocs and fixed a few methods that weren't in JDocs notations (mainly all the binding functions)

### CODE REVIEW

File: src/vertex-array.js [hunk 1]
```diff
@@ -169,7 +177,12 @@ class VertexArray {
         return this;
     }
 
-    // Bind this vertex array.
+    /**
```
Comment (tsherif) [inline src/vertex-array.js]: Shouldn't be in the docs. Same reason as above.

File: src/vertex-array.js [hunk 2]
```diff
@@ -179,7 +192,12 @@ class VertexArray {
         return this;
     }
 
-    // Attach an attribute buffer
+    /**
```
Comment (tsherif) [inline src/vertex-array.js]: Shouldn't be in the docs. Same reason as above.

**Comment by sjfricke** [comment]
Would you suggest using [@ignore](http://usejsdoc.org/tags-ignore.html) instead then.

Or maybe put something like

`// [Private] Attach an attribute buffer`

**Comment by tsherif** [comment]
Sure, as long as they don't appear in the JSDoc documentation. You can run `npm run docs` to regenerate the docs and check.

**Comment by sjfricke** [comment]
The `@ignore` works, but figured with #92 the master needs a docs and build update together

**Comment by tsherif** [comment]
Very nice. Thanks!

**Pull request merged**